tests/baseline/cases/staged-simple-ami-bc.yml

```diff
@@ -21,10 +21,12 @@ items:
     - dest1
     ena_support: true
     image_id: null
+    marketplace_entity_type: AMIProduct
```

Comment:
Right now there are two tests loading AMI items through 'staged' source.  You've updated both of them to use the new fields at non-default values.

Can we please instead have:
1. at least one test which specifies the new fields / loads them at non-default values, and
2. at least one test which does not specify any of the new fields (thus applying the default values)

All tools which exist today for generating the staging areas will generate them like (2) so it's important we don't break that case, and therefore important we test it.

Note, you can always add some more cases rather than trying to cram it all into the two existing cases.

Reply:
I have removed newly added metadata from simple-ami